apps/style/applab/style.scss

```diff
@@ -239,9 +239,10 @@ div#visualizationResizeBar {
   // make sure to update the AdvancedShareOptions.jsx iframe code.
   width: 352px;
   height: 612px;
-  margin: 0 auto;
   padding: 20px 0;
-
+  position: fixed;
+  left: 50%;
+  margin-left: -178px;
```

Comment:
nit: why -178 here? add a comment explaining the value

Reply:
This is good feedback because magic numbers are bad practice and also because I had done my math wrong and was off by two pixels. :laughing:  Updated now!